components/proposal-progress.tsx

```diff
@@ -1,40 +1,97 @@
 import React from "react";
 
-import { formatNumberToString } from "helpers/formatNumber";
+import { useTranslation } from "next-i18next";
 
-import { DistribuitonPerUser } from "interfaces/proposal";
+import { formatNumberToString } from "helpers/formatNumber";
 
+import { DistributedAmounts } from "interfaces/proposal";
 
 import Avatar from "./avatar";
+import InfoTooltip from "./info-tooltip";
 
 interface IProposalProgressProps {
-  usersDistribution: DistribuitonPerUser[];
+  distributedAmounts: DistributedAmounts;
+}
+
+function ProgressItem({
+  percentage,
+  githubLogin = null,
+  label = "",
+  description = "",
+  progressColor = "primary",
+}) {
+  return (
+    <div
+      className={
+        "rounded-bottom d-flex flex-column align-self-end align-items-center gap-2 min-w-fit"
+      }
+      style={{ width: `${percentage}%` }}
+    >
+      {githubLogin ? (
```

Comment:
guess we shouldn't have deleted that GithubAvatar component heh? [re-introduce that component and use it where necessary in this PR]